assets/index.html

```diff
@@ -29,13 +29,6 @@
   <script src="js/popper.min.js"></script>
   <script src="js/bootstrap.bundle.min.js"></script>
```

Comment:
Bootstrap und Popper analog vue und axios?

Reply:
Ja, könnte man auch jetzt schon zu einer Dependency machen. Hatte das ja in "know issues" auch erwähnt jQuery würd ich aber erstmal als Script lassen und dann später durch https://bootstrap-vue.org ersetzen.
Ich update den PR heute Abend.

Reply:
Îm Prinzip könnten wir‘s ja auch lassen. Bootstrap 5 steht schon vor der Tür, ggf. Lohnt es sich auch nicht hier zu investieren. Mit bootstrap-vue habe ich mich noch nicht auseinander gesetzt, es schien mir aber durchaus komplex und eben deutlich weniger dokumentiert als Bopotstrap selbst. Würde uns das wirklich helfen?

Reply:
Ich habe bootstrap-vue auch noch nicht selbst verwendet. Der große Vorteil ist halt, dass man dann klar beim deklarativem Paradigma von Vue bleibt und es nicht mit dem imperativen von jQuery mischt. Aber ich habe den bisherigen Frontend Code noch nicht genau genug gelesen/verstanden. Daher würde ich diese Entscheidung auch gerne erstmal verschieben.